source/fundamentals/time-series.txt

```diff
@@ -28,7 +28,7 @@ using the MongoDB Shell, see our
 
 .. .. important:: Use the MongoDB Shell For Administrative Commands
```

Comment:
S: uncomment this admonition

Reply:
I'm going to leave this commented out based on the reasoning for commenting this out in the first place:
https://github.com/mongodb/docs-node/pull/242#discussion_r704456108
https://jira.mongodb.org/browse/DOCSP-18605

Basically, we don't outright tell people not to use a functionality that is included in the driver, but you can see in the preceding paragraph that it's recommended (with less severe language) that the shell be used instead.